### ISSUE TITLE
The process stops

### ISSUE THREAD

**Issue by ahkaroly** [redphx/roborock-tuya-token#1]
Hello,

Do you have any idea what the problem might be?

That's all that happens. I'm doing it with a rooted NOX emulator.

![roborock](https://user-images.githubusercontent.com/11458787/110984362-52462480-836b-11eb-8407-da33eb0d5ddc.PNG)

**Comment by redphx** [comment]
Did you log in after that? Is your account in EU?

**Comment by ahkaroly** [comment]
Yes I logged in. I have an EU account. I just tried it with an old phone, but that's the same mistake.

**Comment by redphx** [comment]
> Yes I logged in. I have an EU account. I just tried it with an old phone, but that's the same mistake.

Could you please switch your account's server to US and try again? My guess is that your account is not migrated to Tuya yet.

**Comment by bbr111** [comment]
Same Problem here, EU-Account.
tried with bluestacks rooted. not logout output

but i created a full log without filter and find this:

Caused by: java.lang.ClassNotFoundException: Didn't find class "com.tuya.sdk.log.LogPlugin" on path: DexPathList[[zip file "/data/app/com.roborock.smart-1/base.apk"],nativeLibraryDirectories=[/data/app/com.roborock.smart-1/lib/arm, /data/app/com.roborock.smart-1/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib, /data/downloads, /data/priv-downloads]]

is com.tuya.sdk.log.LogPlugin used for writing the log informatation?

**Comment by redphx** [comment]
No, I used `Log.i()` to print the token.  
I see most EU accounts having this same problem. Maybe switching region/create a new account might work.

**Comment by iceman73** [comment]
I am having the same issue and I am in a US region.

**Comment by u20p17** [comment]
Hi,
I also have the same issue. Bluestack 4 rooted with the roborock_2.3.32_token.apk installed and European Server. Any idea how we can fix this?
It stops here:
![image](https://user-images.githubusercontent.com/63282286/118561317-1b293080-b76b-11eb-9ac1-2f511a1627ea.png)

**Comment by new-kirte** [comment]
I am having same issue, UAE

**Comment by Xyaren** [comment]
Version 3.0.0 Of the app is available, would you be so kind to release a modified version of that aswell ?

**Comment by Zer0x00** [comment]
Same problem here with an EU account.. any updates @redphx ?

**Comment by jappish84** [comment]
I'm having the same issue, I'm stuck at:

```
./adb logcat -s "ROBOROCK-name", "ROBOROCK-devId", "ROBOROCK-localKey"

--------- beginning of crash

--------- beginning of system

--------- beginning of main
```


Can't see how I can change region. I have however tried running both modded apps, and running adb on both windows 10 and linux, the results are the same

EDIT:

I have copied the rr_tuya_(number) file from a Android emulator running the modded roborock app. Can I somehow open this file and find the token manually?


EDIT2:

I have now also tried creating a new account and selecting USA as region and readded the vac to the modded app/account and I'm still stuck at the same point in the process.

**Comment by mercenaruss** [comment]
Exactly the same story, Region MD, modded apk, nothing come up.
